.github/workflows/ci_testing.yml

```diff
@@ -57,3 +57,36 @@ jobs:
           fail_ci_if_error: true
           verbose: true
         if: matrix.os == 'ubuntu-latest' && matrix.python-version == '3.9'
+
+  Build-and-Test-Win:
+    runs-on: windows-2019
+    strategy:
+      matrix:
+        python-version: ["3.8", "3.9", "3.10", "3.11"]
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: recursive
+      - uses: conda-incubator/setup-miniconda@v2
+        with:
+          miniconda-version: "latest"
+          activate-environment: ale
+          environment-file: environment.yml
+          auto-activate-base: false
+          auto-update-conda: true
+          python-version: ${{ matrix.python-version }}
+      - name: Check build environment
+        run: |
+          conda list
+      - name: Build Package
+        run: |
+          mkdir -p build
+          cd build
+          cmake -DCMAKE_BUILD_TYPE=RELEASE -DCMAKE_WINDOWS_EXPORT_ALL_SYMBOLS=ON -G "Visual Studio 16 2019" -A x64 -DALE_BUILD_TESTS=OFF ..
```

Comment:
Does the build machine have this specific VS Studio?

Reply:
See: https://github.com/DOI-USGS/ale/actions/runs/5468659762/jobs/9956589863#step:5:19

Reply:
I think so? This is the same version that was being used in the appveyor build and is the same version that USGSCSM uses